Source/WebCore/platform/graphics/texmap/TextureMapperShaderProgram.cpp

```diff
@@ -151,6 +151,7 @@ static const char* vertexTemplateCommon =
     GLSL_DIRECTIVE(ifdef ENABLE_TextureExternalOES) \
         GLSL_DIRECTIVE(extension GL_OES_EGL_image_external : require) \
         GLSL_DIRECTIVE(define SamplerExternalOESType samplerExternalOES) \
+        STRINGIFY(precision mediump samplerExternalOES;\n) \
```

Comment:
is there any reason why you're not using GLSL_DIRECTIVE as the rest of the code?

Reply:
Yes, GLSL_DIRECTIVE  adds # to the beginning of the line.
With GLSL_DIRECTIVE line would become
`#precision mediump samplerExternalOES;`
and that's not valid shader code

Reply:
ah, you're right! But please, follow the same format as the rest of the file:
STRINGIFY(
    precision mediump samplerExternalOES;
)

Reply:
It's inside a macro so it would be a slightly different
```
STRINGIFY( \
    precision mediump samplerExternalOES;\n \
) \
```
I think one-liner is more clean looking, WDYT?

Reply:
ok, updated formatting accordingly